examples/trials/auto-gbdt/config.yml

```diff
@@ -0,0 +1,20 @@
+authorName: default
+experimentName: example_auto-gbdt
+trialConcurrency: 1
+maxExecDuration: 10h
+maxTrialNum: 1

```

Comment:
could we set maxTrialNum 5 or 10, because some users complain that it is a little confusing that the experiment stopped after one trial job is completed.